hack/ci/gardener-integration-aws-specific.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
We have already a script for that in tests/integration/scripts. I assume this is a way of duplicating scripts so prow won't fail after merging?